extensions/spotify-player/package.json

```diff
@@ -99,132 +104,179 @@
       "mode": "no-view",
       "disabledByDefault": true
     },
     {
       "name": "volume25",
       "title": "Set Volume to 25%",
       "subtitle": "Spotify",
-      "description": "Use this to set the volume to 25%.",
+      "description": "Set the volume to 25%.",
       "mode": "no-view",
       "disabledByDefault": true
     },
     {
       "name": "volume50",
       "title": "Set Volume to 50%",
       "subtitle": "Spotify",
-      "description": "Use this to set the volume to 50%.",
+      "description": "Set the volume to 50%.",
       "mode": "no-view",
       "disabledByDefault": true
     },
     {
       "name": "volume75",
       "title": "Set Volume to 75%",
       "subtitle": "Spotify",
-      "description": "Use this to set the volume to 75%.",
+      "description": "Set the volume to 75%.",
       "mode": "no-view",
       "disabledByDefault": true
     },
     {
       "name": "volume100",
       "title": "Set Volume to 100%",
       "subtitle": "Spotify",
-      "description": "Use this to set the volume to 100%.",
+      "description": "Set the volume to 100%.",
       "mode": "no-view",
       "disabledByDefault": true
     },
     {
       "name": "volumeDown",
       "title": "Turn Volume Down",
       "subtitle": "Spotify",
-      "description": "Use this to turn the volume down by 10%.",
+      "description": "Turn the volume down by 10%.",
       "mode": "no-view",
       "disabledByDefault": true
     },
     {
       "name": "volumeUp",
       "title": "Turn Volume Up",
       "subtitle": "Spotify",
-      "description": "Use this to turn the volume up by 10%.",
+      "description": "Turn the volume up by 10%.",
       "mode": "no-view",
       "disabledByDefault": true
     },
     {
       "name": "toggleShuffle",
       "title": "Toggle Shuffle",
       "subtitle": "Spotify",
-      "description": "Use this to toggle shuffle.",
+      "description": "Toggle shuffle.",
       "mode": "no-view",
       "disabledByDefault": true
     },
     {
-      "name": "toggleRepeat",
-      "title": "Toggle Repeat",
+      "name": "cycleRepeat",
+      "title": "Cycle Repeat",
       "subtitle": "Spotify",
-      "description": "Use this to toggle repeat.",
+      "description": "Cycle the repeat state between off, context (meaning the current playlist/album), and track.",
       "mode": "no-view",
       "disabledByDefault": true
     },
+    {
+      "name": "replay",
+      "title": "Replay",
+      "subtitle": "Spotify",
+      "description": "Replay the current song/episode form the beginning.",
+      "mode": "no-view",
+      "disabledByDefault": true
+    },
+    {
+      "name": "seek",
+      "title": "Seek",
+      "subtitle": "Spotify",
+      "description": "Seek to a position in the current song/episode.",
+      "mode": "no-view",
+      "disabledByDefault": true,
+      "arguments": [
+        {
+          "name": "position",
+          "placeholder": "Position",
```

Comment:
On a second thought I dont think we need this. Is this something you'd personally use? I cant imagine this being useful. On a track, it makes more sense, because the position is relative. But as an argument its way too arbitrary imo

Reply:
How about if I add an option to seek forward/backward with +/-, like `+10` to seek forward 10 seconds, and `-5` to go backwards 5 seconds?

Reply:
I think that's better, it's stick with just +10 and -10 though

Reply:
I've made the command now only seek forward/backwards. `10` would seek forward, same as `+10`, while `-10` would go back

Reply:
Removed the seek command entirely after discussion on Slack because it wasn't useful enough